Neuro/Minigames/Solvers/CommsSabotagedSolver.cs

```diff
@@ -16,26 +16,37 @@ public IEnumerator CompleteMinigame(TuneRadioMinigame minigame)
         yield return InGameCursor.Instance.CoMoveToPositionOnCircle(minigame.dial.DialTrans, radius, 90f);
         InGameCursor.Instance.StartHoldingLMB(minigame);
 
-        // check the left side of the dial
-        for (float t = 0; t < 1f; t += Time.deltaTime)
+        // if target angle is positive the signal is on the left
+        if (minigame.targetAngle > 0)
```

Comment:
I don't like the idea of always going in the right direction, it doesn't feel very natural, i think it should search both ways. Keep in mind that theoretically we're not supposed to know where the target is when solving this minigame

Reply:
Updated to add a random search. (50% chance to search slightly in the incorrect direction)